src/hooks/useSessionCamera.ts

```diff
@@ -0,0 +1,51 @@
+import * as THREE from "three"
+
+const CAMERA_KEY = "cadViewerCameraStateSession"
+
+// ✅ Save camera
+export const saveCameraToSession = (camera: THREE.Camera, controls: any) => {
+  try {
+    const data = {
```

Comment:
data is a banned word

```suggestion
    const savedCameraSession = {
```